test/js/dropdown-utils_test.js

```diff
@@ -38,6 +38,40 @@ describe('Dropdown utils', function() {
     expect($('option')).to.have.length(4);
   });
 
+  it('should not add options with same values', function() {
+    dd('foo').addOption({label: 'first', value: 'bar'});
+    dd('foo').addOption({label: 'second', value: 'bar'});
+    expect($('option')).to.have.length(2);
+  });
+
+  it('should select the option when values.select', function() {
+    dd('foo').addOption({label: 'Foo', value: 'BAR', select: 1});
+    expect( $('#foo')[0].selectedIndex ).to.equal(1);
+  });
+
+  it('should create option with value/name of ""', function() {
+    dd('foo').addOption(null);
+    $('#foo')[0].selectedIndex = 1;
+    expect($('#foo').val()).to.equal('');
+  });
+
+  it('should disable several options', function() {
```

Comment:
I was thinking in terms of you are disabling value1 and value2, and those are your selected options to disable, not necessary highlighting types of selecting.

When typing descriptions, I try to use the rules of what scenarios I am testing in this test here, what am I manipulating and what results I am expecting from it.  So I think this description here just outlines in terms of the functionalities this module provides but not the scenario it is trying to test, so it gets a bit broad.

Nothing wrong or right, just thought maybe it will provide a bit more details when reading :)